lib/shared/config-validator.js

```diff
@@ -214,6 +214,19 @@ function validateGlobals(globalsConfig, source = null) {
         });
 }
 
+/**
+ * Validate `processor` configuration.
+ * @param {string|undefined} processor The processor name.
+ * @param {string} source The name of config file.
+ * @param {function(id:string): Processor} getProcessor The getter of defined processors.
+ * @returns {void}
+ */
+function validateProcessor(processor, source, getProcessor) {
```

Comment:
I would really like the first parameter to be `processorName`, for clarity. (I realize the JSDoc makes this clear, but looking at this line in isolation, I could get confused into thinking `getProcessor` is redundant since we have a `processor` already.)